psydac/polar/c1_linops.py

```diff
@@ -280,8 +280,8 @@ def toarray( self ):
         return a
 
     # ...
-    def tosparse( self ):
-        return self.tocoo()
+    def tosparse( self, *, order='C' ):
+        return self.tocoo(order=order)
```

Comment:
The `order` argument is not used, plus I cannot see its meaning for a COO matrix

Reply:
same as above